src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -56,6 +56,11 @@ foam.CLASS({
       name: 'browseTitle',
       expression: function(of) { return foam.String.pluralize(of.model_.label); }
     },
+    {
+      class: 'String',
+      name: 'browseSubtitle',
+      expression: function(of) { return "View all " + foam.String.pluralize(of.model_.label.toLowerCase()) +'.' }
```

Comment:
There is a 'plural' property on Model, which you could/should use instead.

Reply:
Fixed.